Buildasaur/MenuItemManager.swift

```diff
@@ -19,13 +19,12 @@ class MenuItemManager : NSObject, NSMenuDelegate {
         
         let statusItem = statusBar.statusItemWithLength(32)
         statusItem.title = ""
-        let image = NSImage(named: "icon")
-        image?.setTemplate(true)
```

Comment:
Yes I set it for icon image as default, so you don't have to set anything in code manually...